src/GraphQL.Conventions/CommonAssemblyInfo.cs

```diff
@@ -8,8 +8,8 @@
 [assembly: AssemblyCopyright("Copyright 2016-2017 Tommy Lillehagen. All rights reserved.")]
 [assembly: AssemblyTrademark("")]
 [assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("2.0.1")]
-[assembly: AssemblyInformationalVersion("2.0.1")]
+[assembly: AssemblyFileVersion("2.0.2")]
```

Comment:
Need to update the project file too:
 * src/GraphQL.Conventions/GraphQL.Conventions.csproj

Also, noticed:
 * samples/DataLoaderWithEFCore/DataLoaderWithEFCore/DataLoaderWithEFCore.csproj
    (this should really have a project reference instead of a NuGet reference - mind fixing up whilst you're at it?)